### ISSUE TITLE
why is Network Manager required for the help message?

### ISSUE THREAD

**Issue by alexxroche** [sdushantha/wifi-password#41]
```
$ wifi-password -h
ERROR: Network Manager is required to run this program on Linux.
```
Do I need to be connected in order to read the help message... in order to understand how to get connected?

**Comment by sdushantha** [comment]
That is really odd, that shouldn't be happening.

After looking through the code, it looks like `get_ssid()` gets executed even though `--ssid` has not been given.
```python
parser.add_argument('--ssid', "-s", default=get_ssid(), help="Specify a SSID that you have previously connected to")
```

I assume this is because it is set to the default value of the `--ssid` argument. I thought providing `--help` to script would halt everything and just print out the help message.

I'll have to remove that function from the default value and alter the code a little so that everything functions as expected.

**Comment by sdushantha** [comment]
Thank you for letting me know about this error. It has not been fixed in a818597.

I wasn't able to catch it since I currently don't use Linux. It is always nice to have users from other platforms test out my projects 😊
